test_notebooks/trajectory_visualization.py

```diff
@@ -198,12 +179,14 @@ def shift_trajectory(xydata):
 
                 justright = False
 
-                return "Array has more than three columns. May not yield correct results"
+                print("Array has more than three columns. May not yield correct results")
 
             else:
 
                 justright = False
-                return "Array doesn't have enough columns"
+                print("Array doesn't have enough columns")
+
+    return (numerical, justright)
 
 
 def plot_trajectory(xydata, charttitle):
```

Comment:
I'm not sure if this will be a static/instantaneous alternative to the eventual animated plot, but if so: without the animation, there's no way to get a sense of the timescale of each movement of the particle.  It might be informative to drop a circle glyph or something every 10 or so timepoints, so the user gets a sense of the speed of the particle by the spacing of the glyphs for each section of the trajectory.